packages/application-shell/package.json

```diff
@@ -73,18 +73,17 @@
     "history": "4.7.2",
     "is-retina": "1.0.3",
     "jwt-decode": "2.2.0",
-    "lodash.camelcase": "4.3.0",
-    "lodash.flatmap": "4.5.0",
-    "lodash.has": "4.5.2",
-    "lodash.isnil": "4.0.0",
-    "lodash.last": "3.0.0",
-    "lodash.omit": "4.5.0",
-    "lodash.once": "4.1.1",
-    "lodash.snakecase": "4.1.1",
+    "lodash.camelcase": "^4.3.0",
```

Comment:
so this leads to no more double bundling of lodash libs?

Reply:
That's what I am trying to solve. Given they update easily in diverging ways here and on the mc-fe I don't see harm in having a range given how mature the library is too.